src/app/forms.py

```diff
@@ -0,0 +1,33 @@
+from django import forms
+from django.core import validators
+from django.contrib.auth.models import User
+
+# Bare bones check that street address starts with a number.
+# ~~ More functionality needed ~~
+def validate_location(value):
+    if (not value[0].isdigit()):
+        raise forms.ValidationError("Please enter a valid address.")
+
+# Errand entry form
+# ~~ validators still need work ~~
+class ErrandForm(forms.Form):
+    title=forms.CharField(min_length=1, max_length=20, strip=True, label="Title",
+        widget=forms.TextInput(attrs={'placeholder':'Grocery Shop', 'style':'width:60px'}))
+    priority=forms.CharField(min_length=0, max_length=1, strip=True, label="Priority",
```

Comment:
Since priority is stored as a `SmallIntegerField` we should probably use ``IntegerField`` to prevent weirdness assigning from one to another.